### DISCUSSION TITLE
rc-v1.4

### DISCUSSION THREAD

**Discussion by GeorgianStan** [GeorgianStan/vanilla-context-menu#34] [Ideas & Suggestions]
Release candidate for nested menu feature (distribution only).

<hr /><em>This discussion was created from the release <a href='https://github.com/GeorgianStan/vanilla-context-menu/releases/tag/rc-v1.4'>rc-v1.4</a>.</em>

**Comment by kla-ko** [comment]
Hi @GeorgianStan ,
Basically works as expected. Two things which IMO should be improved:
- A ``MenuItem`` with the ``MenuOption`` ``nestedMenu`` should always have ``preventCloseOnClick: true``
- Hovering over a submenu parent should expand the submenu as well. Maybe this behavior can be configured by an option. Still, the default should be "open on hover".

Regards

**Reply by GeorgianStan** [reply]
Hi @kla-ko! Thank you for your feedback. I'll try to implement this in a future release.

**Comment by leonard2z** [comment]
NestedMenu is awesome!
